airflow/providers/apache/spark/hooks/spark_submit.py

```diff
@@ -208,6 +208,9 @@ def _resolve_connection(self):
                 self._conn_id, conn_data['master']
             )
 
+        if 'spark.kubernetes.namespace' in self._conf:
+            conn_data['namespace'] = self._conf['spark.kubernetes.namespace']
```

Comment:
Maybe better change line 204, WDYT
```py
conn_data['namespace'] = self._conf['spark.kubernetes.namespace'] if 'spark.kubernetes.namespace' in self._conf else extra.get('namespace')
```

Reply:
It's too long a line, so we'd have to wrap it anyway, at which point the benefit of using a ternary operator goes away.

Reply:
Get it